mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -131,11 +127,11 @@ impl ConnectionSettings {
         &self.credentials
     }
 
-    pub fn subscriptions(&self) -> &Vec<Subscription> {
+    pub fn subscriptions(&self) -> &Vec<Topic> {
         &self.subscriptions
     }
 
-    pub fn forwards(&self) -> &Vec<Forward> {
+    pub fn forwards(&self) -> &Vec<Topic> {
         &self.forwards
     }
 
```

Comment:
```rust
pub fn keep_alive(&self) -> Duration
```
we can remove `&` symbol here and get rid of `*` wherever we use this method. It works because `Duration` implements `Copy` trait which is fast cloning.